py3-h5py.yaml

```diff
@@ -3,25 +3,40 @@ package:
   description: 'Read and write HDF5 files from Python'
   url: 'https://www.h5py.org'
   version: 3.12.1
-  epoch: 0
+  epoch: 1
   copyright:
     - license: BSD-3-Clause
+  dependencies:
+    provider-priority: 0
+
+vars:
+  pypi-package: h5py
+
+data:
+  - name: py-versions
+    items:
+      3.10: "310"
+      3.11: "311"
+      3.12: "312"
 
 environment:
   contents:
     packages:
       - build-base
       - busybox
-      - cython~0
       - hdf5-dev
       - liblzf-dev
       - numpy
-      - py3-gpep517
-      - py3-pkgconfig
-      - py3-setuptools
-      - py3-wheel
-      - python3-dev
+      - py3-supported-cython
+      - py3-supported-numpy
+      - py3-supported-pip
+      - py3-supported-pkgconfig
+      - py3-supported-python-dev
+      - py3-supported-tomli
       - wolfi-baselayout
+  environment:
+    H5PY_SYSTEM_LZF: 1
+    H5PY_SETUP_REQUIRES:
```

Comment:
setting this explicitly to '0' seems more obvious than haivng an empty string .
if you wanted/needed to set it to an empty string could you _set_ it to an empty string? or maybe I just don't understand what this does here.